src/btreemap.rs

```diff
@@ -2317,6 +2317,21 @@ mod test {
         });
     }
 
+    #[test]
+    fn iter_count_test() {
```

Comment:
proptests are almost always preferable to standard unit tests :)

Reply:
Done.